connectors/perisonic.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+Connector.artistSelector = '#title h3';
+
+Connector.trackSelector = '#title p';
+
+Connector.trackArtSelector = '#coverart img';
+
+Connector.isPlaying = () => $('#playpause').hasClass('pause');
+
+$('#title').bind("DOMSubtreeModified", Connector.onStateChanged);
```

Comment:
I guess the `#title` element contains track info only. If I am correct, the extension won't process play/pause state change.

Is there any element which contains both track info and play/pause button? If so, this element can be used as a player selector (`Connector.playerSelector`).

Reply:
> I guess the #title element contains track info only. If I am correct, the extension won't process play/pause state change.

Yes, it is not able to process play/pause. Currently, it doesn't feel like a limitation.

> Is there any element which contains both track info and play/pause button? If so, this element can be used as a player selector (Connector.playerSelector).

I tried that element and it is not working.

If you want to give me a better advice or point me to a more detailed docu for a such a player situation I would be satisfied.

Reply:
I cannot test the connector, because Perisonic doesn't work for me (Subsonic web UI works fine, though). But I looked into page source and noticed a `#player` element which contains player elements and track info. Can this element be used as a player selector? If not, I'll accept this PR anyway, since it works for you fine.

Reply:
this element didn't work

Reply:
That's sad.